cmd/juno/juno.go

```diff
@@ -68,6 +69,7 @@ const (
 	defaultMetricsPort         = 9090
 	defaultGRPC                = false
 	defaultGRPCPort            = 6064
+	defaultRPCMaxConcurrency   = 1024
```

Comment:
Is `1024` better than `2 * runtime.GOMAXPROCS(0)`?

Reply:
1024 is practically unlimited concurrency. Users can decide to limit if they want to.

@wojciechos was seeing `broken pipe` errors in load tests. I am assuming those were related to `2 * runtime.GOMAXPROCS(0)` being too low